packages/codebytes/src/editor.tsx

```diff
@@ -60,7 +62,7 @@ export const Editor: React.FC<EditorProps> = ({
         .writeText(text)
         // eslint-disable-next-line no-console
         .catch(() => console.error('Failed to copy'));
-      onCopy?.(text, language);
+      on?.copy?.(text, language);
```

Comment:
we could default to an empty object cast as CodebytesChangeHandlerMap to avoid all the conditional chaining

Reply:
this is totally sound 💯